Sources/TuistKit/Commands/TuistCommand.swift

```diff
@@ -155,6 +152,7 @@ public struct TuistCommand: AsyncParsableCommand {
     }
 
     private static func outputCompletion(logFilePath: AbsolutePath, shouldOutputLogFilePath: Bool) {
+        print("", terminator: "\n")
```

Comment:
The terminator looks slightly weird when generating the project. fwiw, the output looks a bit better before using Noora in this case, but we can keep it as-is as that will improve once all the `generate` output will be done via Noora components:
<img width="978" alt="image" src="https://github.com/user-attachments/assets/1f17c9bb-18d4-49ff-887b-bc4119c8997f" />

Reply:
I ended up leaving it to create some breath between the two sections. I think once we update the other outputs, it'll look much better.